epochplugins/cluster.py

```diff
@@ -17,8 +21,60 @@ def populate_options(self, epoch_client: epochclient.EpochClient, subparser: arg
         sub_parser = commands.add_parser("leader", help="Get the leader on the cluster")
         sub_parser.set_defaults(func=self.leader)
 
+        sub_parser = commands.add_parser("save", help="save the topologies in a json")
+        sub_parser.add_argument("file_name", metavar="file-name", help="Name of file to be saved")
+        sub_parser.set_defaults(func=self.save)
+
+        sub_parser = commands.add_parser("load", help="load the topologies to the given cluster")
+        sub_parser.add_argument("-file_name", metavar="file-name", help="Name of file to be loaded")
+        sub_parser.add_argument("--override", metavar="override_flag",
+                                help="For controlling overriding of the existing topologies", default=False)
+        sub_parser.add_argument("--paused", metavar="paused", help="Load all paused topologies", default=False)
+        sub_parser.add_argument("--skip", metavar="skipped topologies", help="Topologies to be skipped via loading",
+                                default=[], nargs="*")
+
+        # sub_parser.add_argument('--override', action='override', help='Override existing topologies if the topology exists')
+        # sub_parser.add_argument("run_id", metavar="run-id", help="Run ID")
+        # sub_parser.add_argument("task_name", metavar="task-name", help="Task Name")
+        sub_parser.set_defaults(func=self.load)
+
         super().populate_options(epoch_client, parser)
 
     def leader(self, options: SimpleNamespace):
         data = self.epoch_client.get("/apis/housekeeping/v1/leader")
         print(data)
+
+    def save(self, options: SimpleNamespace):
+        data = self.epoch_client.get("/apis/v1/topologies")
+        file_name = options.file_name
+        with open(file_name + ".json", 'w') as f:
+            json.dump(data, f)
+
+    def load(self, options: SimpleNamespace):
+        current_cluster_data = self.epoch_client.get("/apis/v1/topologies")
+        json_data = epochutils.load_json(options.file_name)
+        topologies_to_load = self.filter_topologies(current_cluster_data, json_data, options)
+        for data in topologies_to_load:
+            try:
+                self.epoch_client.post("/apis/v1/topologies", data.get("topology"))
+                print("Topology created : {topology_id}".format(topology_id=data.get("name")))
+            except Exception as ex:
+                print("Error creating topology. Error: " + str(ex))
+
+    def filter_topologies(self, current_cluster_data, topologies_from_json, options: SimpleNamespace):
+        if len(options.skip) > 0:
+            topologies_from_json = [topology for topology in topologies_from_json if
+                                    topology.get('id') not in options.skip]
+        if options.paused:
```

Comment:
This would change based on the above comment. Might require making the PAUSE api call after creating the topology, or changing the json itself, before creating the topology